wye/templates/email_messages/contactus/message_user.html

```diff
@@ -0,0 +1,7 @@
+Hey {{contact_name}},
```

Comment:
@harisibrahimkv Can you send pull request with better email message for this context ?